static/app/views/replays/details_v2.tsx

```diff
@@ -63,7 +65,11 @@ function ReplayDetails() {
   return (
     <Page eventSlug={eventSlug} orgId={orgId} event={replay?.getEvent()}>
       <ReplayContextProvider replay={replay} initialTimeOffset={initialTimeOffset}>
-        <Layout layout={getParamValue() === 'sidebar' ? 'sidebar' : 'topbar'} />
+        <Layout
+          layout={
+            LAYOUT_NAMES.includes(getParamValue()) ? (getParamValue() as any) : 'topbar'
```

Comment:
Lets timebox fixing this, if it's more annoying or takes longer than 20mins add a TODO comment instead.

The pulse right now is that we want only one layout in the end, so I'm expecting that this prop will not be needed after we delete the two extra layout options.